files/en-us/web/css/css_anchor_positioning/try_options_hiding/index.md

```diff
@@ -98,26 +100,22 @@ The anchor-positioned element is given fixed positioning and tethered to the anc
 }
 ```
 
-> **Note:** When multiple try options are specified, they are separated by commas. Position try options are tried in the order they are specified in.
-
 Try scrolling the demo so that the anchor starts to get near the edges:
 
 {{ EmbedLiveSample("Using predefined try options", "100%", "250") }}
 
-- Move the anchor to the top of the viewport. The positioned element flips to the bottom-left of the anchor to avoid overflowing.
-- Move the anchor to the left of the viewport. The positioned element flips to the top-right of the anchor to avoid overflowing.
+- Keep the anchor horizontally centered, and move the anchor to the top of the viewport. The positioned element flips to the bottom-left of the anchor to avoid overflowing.
```

Comment:
if these don't stay centered you get the "ooops" we explain next. Prevents the oops too early.